src/main/java/frc/robot/subsystems/GamePieceManipulatorSubsystem.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot.subsystems;
+
+import static frc.robot.Constants.GamePieceManipulatorConstants.DEVICE_ID_CANrange;
+import static frc.robot.Constants.GamePieceManipulatorConstants.DEVICE_ID_MANIPULATORMOTOR;
+import static frc.robot.Constants.GamePieceManipulatorConstants.IntakeSpeed;
+import static frc.robot.Constants.GamePieceManipulatorConstants.OuttakeSpeed;
+import static frc.robot.Constants.GamePieceManipulatorConstants.ScoreSpeed;
+
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.controls.VelocityTorqueCurrentFOC;
+import com.ctre.phoenix6.hardware.CANrange;
+import com.ctre.phoenix6.hardware.TalonFX;
+import edu.wpi.first.units.measure.Distance;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class GamePieceManipulatorSubsystem extends SubsystemBase {
+  /** Creates a new ExampleSubsystem. */
```

Comment:
This comment looks like it should be on the constructor, and it should be updated to not reference the example

Reply:
This still needs to be addressed